ghost/i18n/locales/hu/ghost.json

```diff
@@ -36,34 +36,34 @@
     "Sent to {email}": "Elküldve ide: {email}",
     "Sign in": "Bejelentkezés",
     "Sign in to {siteTitle}": "Bejelentkezem ide: {siteTitle}",
-    "Sign in verification": "",
+    "Sign in verification": "Bejelentkezési ellenőrzés",
     "Subscription details": "Előfizetés részletei",
-    "Tap the link below to complete the signup process for {siteTitle}, and be automatically signed in:": "Kattints az alábbi linkre a regisztráció befejezéséhez és az automatikus bejelentkezéshez:",
+    "Tap the link below to complete the signup process for {siteTitle}, and be automatically signed in:": "Kattints az alábbi linkre a regisztráció befejezéséhez {siteTitle} oldalnál és az automatikus bejelentkezéshez:",
```

Comment:
AI is flagging this as incorrect case usage, preferring {siteTitle} oldalra és automatikusan bejelentkezel